docs/commands/browser/runStep.mdx

```diff
@@ -0,0 +1,67 @@
+# runStep
+
+## Обзор {#overview}
+
+Используйте команду `runStep`, чтобы получить человекочитаемую историю выполнения теста, которая, в том числе, автоматически будет отображаться в [html-reporter][reporter].
+Шаги могут быть вложенными.
+
+## Использование {#usage}
+
+```javascript
+await browser.runStep(stepName, stepCb);
+```
+
+## Параметры команды {#parameters}
+
+<table>
+<thead>
+<tr><td>**Имя**</td><td>**Тип**</td><td>**Описание**</td></tr>
+</thead>
+<tbody>
+<tr><td>stepName</td><td>String</td><td>Название шага.</td></tr>
+<tr><td>stepCb</td><td>Function</td><td>Функция с набором команд, которые нужно объединить в единый шаг.</td></tr>
+
+</tbody>
+</table>
+
+## Примеры использования {#examples}
+
+```javascript
```

Comment:
Optionally, we can move large enough examples to partial files and import them in ru/en from one place (see my PR). But that's not critical to me, we can keep it as is.